config.js

```diff
@@ -829,6 +829,7 @@ config.NSFS_CONTENT_DIRECTORY_VERSIONING_ENABLED = false;
 config.NSFS_EXIT_EVENTS_TIME_FRAME_MIN = 24 * 60; // per day
 config.NSFS_MAX_EXIT_EVENTS_PER_TIME_FRAME = 10; // allow max 10 failed forks per day
 
+config.GPFS_DL_PATH = '/usr/lpp/mmfs/lib/libgpfs.so';
```

Comment:
It means that you're changing the default, right?
I think you need to update in the docs [here](https://github.com/noobaa/noobaa-core/blob/master/docs/NooBaaNonContainerized/ConfigFileCustomizations.md#7-gpfs-library-path--).